packages/app/src/specs/banners/ComponentTestingAvailableBanner.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <TrackedBanner
+    :banner-id="bannerId"
+    data-cy="component-testing-banner"
+    status="promo"
+    :title="title"
+    class="mb-[16px]"
+    dismissible
+    :icon="iconFromType"
```

Comment:
The icon is upside down.

<img width="254" alt="image" src="https://user-images.githubusercontent.com/1702361/236201409-4b233d6f-c00f-43c1-9ebc-d6acb4fb3d04.png">

Reply:
Blerg, not sure how I missed that. Good 👀 

Looks like `Alert` is treating the prefix icon as a "collapsible" indicator that rotates 180 degrees based on open/close state. Will have to noodle on this one a little bit